src/lib/pickles/cache.ml

```diff
@@ -10,17 +10,19 @@ module Step = struct
         * Backend.Tick.R1CS_constraint_system.t
 
       let to_string : t -> _ = function
-        | _id, header, n, h ->
+        | _id, header, n, _h ->
             sprintf !"step-%s-%s-%d-%s" header.kind.type_ header.kind.identifier
               n header.identifying_hash
     end
 
     module Verification = struct
+      [@@@warning "-4"]
```

Comment:
Don't you need to turn the warning back on again after the offending statement?

Reply:
Changed to the module's scope